cluster/ovm-pvc.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: OfflineVirtualMachine
+metadata:
+  name: my-vm
+  labels:
+    kubevirt.io/my: label
+spec:
+  running: false 
+  template:
+    metadata:
+      labels:
+        kubevirt.io/my: label
+    spec:
+      domain:
+        resources:
+          requests:
+            memory: 64M
+        devices:
+          disks:
+            - name: pvcvolume
+              volumeName: pvcvolume
+              disk:
+                bus: virtio
+            - name: pvcdata
+              volumeName: pvcdata
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              volumeName: cloudinitvolume
+              disk:
+                bus: virtio
+      volumes:
+        - name: pvcvolume
+          persistentVolumeClaim:
+            claimName: disk-alpine
+        - name: pvcdata
+          persistentVolumeClaim:
+            claimName: disk-custom 
+        - name: cloudinitvolume
+          cloudInitNoCloud:
+            userDataBase64: SGkuXG4=
```

Comment:
Just curious, can this really be anything at all like that?

Reply:
It can. This leads to error and fail. Overall this leads to faster boot times, because cloudinit does not try to boot from not existing image.